pkgs/games/gimx/default.nix

```diff
@@ -1,13 +1,16 @@
-{ stdenv, lib, fetchFromGitHub, fetchpatch
-, makeWrapper, curl, libusb1, xorg, libxml2
-, ncurses5, bluez, libmhash, gimxPdpGamepad ? false }:
+{ stdenv, lib, fetchFromGitHub, fetchpatch, makeWrapper, curl, libusb1, bluez, libxml2, ncurses5, libmhash, xorg }:
 
 let
+  gimx-pdp = false;
```

Comment:
If this is not configurable, we might as well drop the code related to it.

Reply:
We may use it in the future.

Reply:
Then this is currently dead code which we don't want to accumulate. We always have the git history to recover it.